src/styles/zendesk.css

```diff
@@ -0,0 +1,70 @@
+#launcher {
```

Comment:
use tailwind classes, especially for the colours

Reply:
@ItsANameToo removed hardcoded values and applied tw classes :+1: